crates/core/src/yarn_fn/function_wrapping.rs

```diff
@@ -132,6 +143,13 @@ impl PartialEq for Box<dyn UntypedYarnFn + Send + Sync> {
 
 impl Eq for Box<dyn UntypedYarnFn + Send + Sync> {}
 
+#[macro_export]
+macro_rules! yarn_fn {
```

Comment:
`yarn_fn!` sounds like a yarn fn is being created, but it's just the signature. How about `yarn_fn_signature!`? Just not `yarn_fn_impl`, because that sounds like we are implementing / creating a yarn fn.

Reply:
Good point. Maybe `impl_yarn_fn` so it reads like `impl YarnFn<...>`?

Reply:
sounds like a name for a macro that takes in some closure and wraps it in a `YarnFn`